### ISSUE TITLE
Versioning conflict on server and client From client: [155006] On server: [155008]

### ISSUE THREAD

**Issue by michaelbukachi** [objecthub-io/ObjectHub#222]
Got this error when doing a reload:
![screenshot from 2017-06-08 20-02-50](https://user-images.githubusercontent.com/10145850/26942681-a6216044-4c8b-11e7-84fb-7510acdff600.png)
However, it was a one time thing though.

**Comment by objecthub-io** [comment]
Thank you for reporting this. So this wasn't caused by doing anything in particular?

In any case, I'll investigate what might have caused this!

**Comment by michaelbukachi** [comment]
Nope. Just tried to reload after an edit and the error showed up.
